app/validators/blocks/calculated_summary_block_validator.py

```diff
@@ -6,6 +6,7 @@ class CalculatedSummaryBlockValidator(CalculationBlockValidator):
         "Answer ids for calculated summary must be set before calculated summary block"
     )
     ANSWER_SET_IN_DIFFERENT_SECTION_FOR_CALCULATED_SUMMARY = "Answer ids for calculated summary must be set in the same section as the calculated summary block"
+    CALCULATED_SUMMARY_WITH_NON_DYNAMIC_SINGLE_ANSWER = "Calculated summaries cannot consist of a single answer unless it is a dynamic answer"
```

Comment:
We can add {answer_id} here so it's easier to track down and fix when validation fails.

Reply:
👍 I added answer_id to the error message in the same way as the block one, so it'd be similar to other CS messages, if that seems OK?